### DISCUSSION TITLE
Test Sirène Heiman HS2WD-E compatible zigbee 3.0 - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#35] [Comments]
# Test Sirène Heiman HS2WD-E compatible zigbee 3.0 - Haade.fr

Sirène intelligente de la marque Heiman HS2WD-E compatible zigbee 3.0, que vaut-elle ?

https://haade.fr/fr/blog/test-alarme-sirene-heiman-zigbee-3-HS2WD-E

**Comment by orcad-fr** [comment]
Merci pour cet article!
J'ai acheté cette sirene pour mon installation mais peu importe le payload que je lui envoie il est impossible de choisir la tonalité ou le volume.... seule la durée est prise en compte.
Comment avez vous réussi ?
